src/server/components/serializers/UtbetalingerSerializer.tsx

```diff
@@ -0,0 +1,195 @@
+  [YtelseType.ORDINÆR_BARNETRYGD]: 'Ordinær barnetrygd',
+  [YtelseType.UTVIDET_BARNETRYGD]: 'Utvidet barnetrygd',
+  [YtelseType.SMÅBARNSTILLEGG]: 'Småbarnstillegg',
+};
+
+const barnetrygdTekst = (fnr: string, ytelseType: YtelseType) => {
+  if (ytelseType == YtelseType.ORDINÆR_BARNETRYGD) {
+    return `Barn ${formatterFnr(fnr)}`;
+  } else {
+    return YtelseTypeText[ytelseType];
+  }
+};
+
+const formatterFnr = (fnr: string) => {
+  if (fnr.length == 11) {
+    return `${fnr.substring(0, 6)} ${fnr.substring(6)}`;
+  }
+  return fnr;
+};
+
+const ZebraStripedTable = styled.table`
+  border: none;
+  width: 100%;
+
+  tbody tr:nth-child(odd) {
+    background-color: #f2f3f5;
+  }
+`;
+
+const CommonCellCSS = css`
+  border: none;
+  display: table-cell;
+  padding: 0.25rem 0.5rem;
+  font-family:
+    Source Sans Pro,
+    sans-serif;
+  font-size: 12pt;
+  letter-spacing: 0;
+  line-height: 1.5rem;
+  margin: 0;
+  border-bottom: 1px solid #838c9a;
+`;
+
+const StyledTableData = styled.td<{ $borderTop?: boolean }>`
+  ${CommonCellCSS};
+  font-weight: 400;
+`;
+
+const StyledTableHeader = styled.th`
+  ${CommonCellCSS};
+  border-bottom: 2px solid #838c9a;
+  font-weight: 600;
+`;
+
+const StyledTableDataRow = styled.tr<{ $borderTop?: boolean }>`
+    ${StyledTableData} {
+        border-top: ${props => (props.$borderTop ? '2px solid #838c9a' : 'none')};
+    }
+}
+`;
+
+const SummaryTableRow = styled.tr`
+  td {
+    font-weight: bold;
+  }
+`;
+
+const UtbetalingerSerializer = (props: UtbetalingerProps) => {
+  const { maalform, utbetalingerPerMndEøs } = props;
+
+  if (!utbetalingerPerMndEøs) {
+    return null;
+  }
+  return (
+    <ZebraStripedTable>
+      <thead>
+        <tr>
+          <StyledTableHeader align="left" scope="col">
+            {TabellHeaderForSpraak[maalform].barnetrygd}
+          </StyledTableHeader>
+          <StyledTableHeader align="left" scope="col">
+            {TabellHeaderForSpraak[maalform].månedÅr}
+          </StyledTableHeader>
+          <StyledTableHeader align="right" scope="col">
+            {TabellHeaderForSpraak[maalform].satsINorge}
+          </StyledTableHeader>
+          <StyledTableHeader align="right" scope="col">
+            {TabellHeaderForSpraak[maalform].utbetaltFraAnnetLand}
+          </StyledTableHeader>
+          <StyledTableHeader align="right" scope="col">
+            {TabellHeaderForSpraak[maalform].utbetaltFraNorge}
+          </StyledTableHeader>
+        </tr>
+      </thead>
+      <tbody>
+        {Object.keys(utbetalingerPerMndEøs).map((mndÅr, mndÅrIndex) => {
+          const utbetalingMndEøs = utbetalingerPerMndEøs[mndÅr];
+          const harFlereYtelserIPeriode = utbetalingMndEøs.utbetalinger.length > 1;
+          return (
+            <Fragment key={mndÅrIndex}>
+              {utbetalingMndEøs.utbetalinger.map((utbetalingEØS, index) => (
+                <StyledTableDataRow
+                  $borderTop={harFlereYtelserIPeriode && index == 0}
```

Comment:
`==` 👀 
Vi ønsker vel å bruke`===`?

https://jsfiddle.net/jsyrnevg/

https://forum.freecodecamp.org/t/difference-between-and/430690/3